staging/src/k8s.io/apiserver/pkg/admission/audit_test.go

```diff
@@ -144,7 +146,13 @@ func TestWithAudit(t *testing.T) {
 		var handler Interface = fakeHandler{tc.admit, tc.admitAnnotations, tc.validate, tc.validateAnnotations, tc.handles}
 		ctx := audit.WithAuditContext(context.Background())
 		ac := audit.AuditContextFrom(ctx)
-		ac.SetEventLevel(auditinternal.LevelMetadata)
+
+		// Set the event level for the test case to work!
```

Comment:
it looks like the event.Level is currently only set in LogRequestMetadata, and is only set based on the RequestAuditConfig already being passed to Init

the last commit of https://github.com/liggitt/kubernetes/commits/eliminate-audit-context-set-event-level/ moves the event level set into Init and lets us Init the test context in these two places instead of needing to use reflect / unsafe